[2022-08-19 21:05:16,018] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2022-08-19T21:00:00+00:00 [queued]>
[2022-08-19 21:05:16,061] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2022-08-19T21:00:00+00:00 [queued]>
[2022-08-19 21:05:16,061] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 21:05:16,061] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 21:05:16,061] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 21:05:16,101] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_aguardando> on 2022-08-19 21:00:00+00:00
[2022-08-19 21:05:16,126] {standard_task_runner.py:52} INFO - Started process 92851 to run task
[2022-08-19 21:05:16,137] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_aguardando', 'scheduled__2022-08-19T21:00:00+00:00', '--job-id', '715', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmpgrpxzp7f', '--error-file', '/tmp/tmpcpmrm59x']
[2022-08-19 21:05:16,138] {standard_task_runner.py:80} INFO - Job 715: Subtask popula_elastic_aguardando
[2022-08-19 21:05:17,381] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_aguardando scheduled__2022-08-19T21:00:00+00:00 [running]> on host 1e1070d9e5ae
[2022-08-19 21:05:17,940] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_aguardando
AIRFLOW_CTX_EXECUTION_DATE=2022-08-19T21:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-19T21:00:00+00:00
[2022-08-19 21:05:17,969] {base.py:271} INFO - GET http://192.168.2.100:9200/ [status:200 request:0.015s]
[2022-08-19 21:05:20,819] {base.py:271} INFO - HEAD http://192.168.2.100:9200/_template/desk_aguardando [status:200 request:0.004s]
[2022-08-19 21:05:20,989] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2022-08-19 21:05:21,193] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.078s]
[2022-08-19 21:05:21,222] {python.py:173} INFO - Done. Returned value was: None
[2022-08-19 21:05:21,296] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_aguardando, execution_date=20220819T210000, start_date=20220819T210516, end_date=20220819T210521
[2022-08-19 21:05:21,468] {local_task_job.py:221} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-08-19 21:05:21,471] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 92851. PIDs of all processes in the group: [92851]
[2022-08-19 21:05:21,471] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 92851
[2022-08-19 21:05:21,508] {process_utils.py:75} INFO - Process psutil.Process(pid=92851, status='terminated', exitcode=0, started='21:05:15') (92851) terminated with exit code 0
